ansible-scylla-node/defaults/main.yml

```diff
@@ -120,7 +120,19 @@ full_inventory: true
 # !!! also make sure you override scylla_yaml_params and remove authorization from there !!!
 scylla_authentication: True
 
+# Set this to false to disable automatic adjustment of system_auth keyspace replication
+adjust_system_auth_replication: False
+
+# Replication variables and cql credentials are intended to be used to adjust system keyspaces when necessary
+# Remember to update cql credentials here in case you're making updates to an already existing cluster which
+# don't have the default username/password(cassandra/cassandra)
+replication_factor: 3
```

Comment:
This variable name is not good - too generic. What if we need to define a replication factor for some other KS(es) in the future?

Something like `scylla_internal_kses_replication_factor` would be better IMO.

Reply:
But what if tomorrow we would want a different config for `audit` and `system_auth`?

So, better define different variables for each.

Reply:
Done